views/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+     <meta name="viewport" content="width=device-width">
+     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+     <script src="https://code.jquery.com/jquery-1.11.2.js"></script>
+     <link href="css/tillScreen.css" rel="stylesheet">
+    <title>The Coffee Connection</title>
+</head>
+
+<body>
+
+<!-- background image -->
+
+<div id="background">
+    <img src='images/till.png' id="backgroundimage" alt='[]' />
+</div>
+
+<!-- results form -->
+
+<section class="numberBox">
+    <section id="errorTill"></section>
+    <section id="tillNumbers"></section>
+    <section id="totalPrice"></section>
+    <section id="tax"></section>
+    <section id="afterTax"></section>
+</section>
+
+<!-- order form -->
+
+<div class="entryBox">
+    <form>
+        <div id="entryTill">
+          <select id="itemselect" class="btn btn-mini">                
+          <optgroup label="Coffee">
+              <option id="4.75">Cafe Latte</option>
+              <option id="4.75">Flat White</option>
+              <option id="3.85">Cappucino</option>
+              <option id="2.05">Single Espresso</option>
+              <option id="3.75">Double Espresso</option>
+              <option id="3.75">Americano</option>
+          </optgroup>
+          <optgroup label="Drinks">
+              <option id="4.55">Cortado</option>
+              <option id="3.65">Tea</option>
+          </optgroup>
+          <optgroup label="Desserts">
+              <option id="6.40">Choc Mudcake</option>
+              <option id="8.20">Choc Mousse</option>
```

Comment:
This is fine but would have been worth considering how you could have generated this data dynamically from the `.json` file